{
    "test_case": "project.add_page_inside_directory",
    "result": "fail",
    "steps": [
        "Navigate to: 'http://localhost:8000'",
        "Write 'admin' in element username",
        "Write 'admin' in element password",
        "Click login_button",
        "Access project test",
        "Write 'yzien/' in element add input",
        "Press key: ENTER",
        "Write 'tgsms' in element add input",
        "Press key: ENTER",
        "error__35abe306"
    ],
    "description": "Verify that the user can create a new page inside a directory",
    "error": "Traceback (most recent call last):\n  File \"/Users/lucianofeo/Desktop/golem/src/golem/test_runner/test_runner.py\", line 115, in run_test\n    test_module.test(execution.data)\n  File \"/Users/lucianofeo/Desktop/golem/example/projects/golem/tests/project/add_page_inside_directory.py\", line 20, in test\n    project.verify_page_exists(data.page_name, data.dir_name)\nTypeError: verify_page_exists() takes 1 positional argument but 2 were given\n",
    "short_error": "TypeError: verify_page_exists() takes 1 positional argument but 2 were given\n",
    "test_elapsed_time": 4.58,
    "test_timestamp": "2017.10.09.18.32.21.452",
    "browser": "chrome",
    "test_data": {
        "env": "{'url': 'http://localhost:8000', 'name': 'test'}",
        "dir_name": "'yzien'",
        "page_name": "'tgsms'"
    },
    "environment": "test"
}